platform/src/core/apis/Account.js

```diff
@@ -8,51 +8,65 @@ import {
   UPDATE_USER_DETAILS_URL,
 } from '../urls/authentication';
 import axios from 'axios';
-import { NEXT_PUBLIC_AUTHORISATION } from '../../lib/envConstants';
+import { NEXT_PUBLIC_API_TOKEN } from '../../lib/envConstants';
 
 let jwtToken;
+let accessToken;
 if (typeof window !== 'undefined') {
   jwtToken = window.localStorage.getItem('token');
-} else {
-  jwtToken = NEXT_PUBLIC_AUTHORISATION;
+} else if (!jwtToken) {
+  accessToken = NEXT_PUBLIC_API_TOKEN;
 }
+
 axios.defaults.headers.common.Authorization = jwtToken;
 
 export const postUserCreationDetails = async (data) =>
-  await axios.post(AUTH_URL, data).then((response) => response.data);
+  await axios
+    .post(AUTH_URL, data, { params: { token: NEXT_PUBLIC_API_TOKEN } })
+    .then((response) => response.data);
 
 export const getGoogleAuthDetails = async () => {
   await axios.get(GOOGLE_AUTH_URL);
 };
 
 export const postUserLoginDetails = async (data) => {
-  return await axios.post(LOGIN_URL, data).then((response) => response.data);
+  return await axios
+    .post(LOGIN_URL, data, { params: { token: NEXT_PUBLIC_API_TOKEN } })
+    .then((response) => response.data);
 };
 
 export const getUserDetails = async (userID) => {
-  return await axios.get(`${USERS_URL}/${userID}`).then((response) => response.data);
+  return await axios
+    .get(`${USERS_URL}/${userID}`, { params: { token: NEXT_PUBLIC_API_TOKEN } })
+    .then((response) => response.data);
 };
 
 export const getAssignedGroupMembers = async (groupID) => {
   return await axios
-    .get(`${GROUPS_URL}/${groupID}/assigned-users`)
+    .get(`${GROUPS_URL}/${groupID}/assigned-users`, { params: { token: NEXT_PUBLIC_API_TOKEN } })
```

Comment:
Hi @Codebmk , why aren't we using the JWT of the user? How shall we be able to know who is accessing what IF all users are utilising the SAME token? cc: @OchiengPaul442 / @FaithDaka